routes/user.php

```diff
@@ -80,3 +80,5 @@
 $this->get('alert/{guid}', ['uses' => 'User\NotificationsController@url']);
 $this->get('notification/{guid}', ['uses' => 'User\NotificationsController@url'])->name('user.notifications.url');
 $this->get('ping', ['uses' => 'User\PingController@index'])->name('ping');
+
+$this->get('session', ['uses' => 'User\SessionTokenController@index'])->name('session');
```

Comment:
Proponowałbym przenieść tę regułę do osobnego pliku - `api.php` (w katalogu `routes`). Wówczas trzeba by było dodać jeszcze plik `api.php` do metody `mapWebRoutes()` pliku `RouteServiceProvider.php`.

Może odpalić tego linka na osobnej subdomenie - np. `api.4programmers.net`? https://laravel.com/docs/5.4/routing#route-group-sub-domain-routing

Wygladaloby to tak:

```php
$this->domain('api.4programmers.net')->group(function () {
    $this->get('session', ['uses' => 'User\SessionTokenController@index'])->name('session');
});
```

Tylko dobrze, aby domena była odczytywana z konfiguracji. Tak więc można przerobić to tak:

```php
$this->domain('api.' . ltrim(config('session.domain'), '.'))->group(function () {
    $this->get('session', ['uses' => 'User\SessionTokenController@index'])->name('session');
});
```

W konfiguracji, w .env trzeba ustawić wówczas domenę

```
SESSION_DOMAIN=.4programmers.net
```